packages/components/src/components/tile/tile.e2e.ts

```diff
@@ -304,7 +304,27 @@ describe("calcite-tile", () => {
           {
             "--calcite-tile-accent-color-press": {
               shadowSelector: `.${CSS.container}`,
-              targetProp: "boxShadow",
+              targetProp: "outline",
```

Comment:
We should switch this back to `boxShadow` for correctness since that is how the "border" option is being applied uniquely for this selection-appearance type.